contrib/cirrus/lib.sh

```diff
@@ -290,3 +290,32 @@ remove_packaged_podman_files() {
     # Be super extra sure and careful vs performant and completely safe
     sync && echo 3 > /proc/sys/vm/drop_caches || true
 }
+
+# Execute make localbenchmarks in $CIRRUS_WORKING_DIR/data
+# for preserving as a task artifact.
+localbenchmarks() {
+    local datadir
+    req_env_vars DISTRO_NV PODBIN_NAME PRIV_NAME TEST_ENVIRON TEST_FLAVOR
+    req_env_vars VM_IMAGE_NAME EC2_INST_TYPE
+
+    datadir=$CIRRUS_WORKING_DIR/data
+    mkdir -p $datadir
+
+    echo "# Env. var basis for benchmarks benchmarks."
+    printenv | grep -Ev "$SECRET_ENV_RE" | sort >> $datadir/benchmarks.env
+
+    (
+      echo "# Machine details for data-comparison sake, not actual env. vars."
+      # Checked above in req_env_vars
+      # shellcheck disable=SC2154
+      echo "\
+CPUTOTAL=$(grep -ce '^processor' /proc/cpuinfo)
+INST_TYPE=$EC2_INST_TYPE  # one day may include other cloud's VM types.
+MEMTOTAL=$(awk -F: '$1 == "MemTotal" { print $2 }' </proc/meminfo | sed -e "s/^ *//")
+UNAME_RM=$(uname -r -m)
+"
+    ) > $datadir/benchmarks.env
+    make localbenchmarks &> $datadir/benchmarks.raw
```

Comment:
It seems likely that the failure is here, and `&>` is hiding the message.

Reply:
Right, and the data collection doesn't happen if the job fails so we can't see it.  Let me fix that in `.cirrus.yml` so we always collect the data, success or not.

Reply:
I'll also change that to a `) |& tee $datadir/benchmarks.raw`

Reply:
rather `make localbenchmarks |& tee $datadir/benchmarks.raw`